applications/AdjointFluidApplication/custom_schemes/adjoint_bossak_scheme.h

```diff
@@ -381,6 +454,14 @@ class AdjointBossakScheme : public Scheme<TSparseSpace, TDenseSpace>
 
         int ThreadId = OpenMPUtils::ThisThread();
 
+        // access the variables through the core for trilinos application.
```

Comment:
it was so I could access variables in adjoint application from trilinos application. after speaking with Jordi I decided to leave the variables in the core for now so I remove this access via kratos components.